src/aui-form-builder/js/aui-form-builder-layout-builder.js

```diff
@@ -421,6 +459,22 @@ A.FormBuilderLayoutBuilder.prototype = {
     },
 
     /**
+     * Fires when click event is triggered.
+     *
+     * @method _onClickOutsideMoveRowTarget
+     * @param {EventFacade} event
+     * @protected
+     */
+    _onClickOutsideMoveRowTarget: function(event) {
+        var targetNode = event.target;
+
+        if (!(targetNode.hasClass(CSS_MOVE_ROW_TARGET))) {
+            this._layoutBuilder.cancelMove();
+            this._detachCancelMoveRowEvents();
```

Comment:
I think this should be called when `layout-builder:moveEnd` is fired instead, otherwise other causes for canceling the move will not detach this event, and so future moves will end up causing multiple events to be attached for this.